### DISCUSSION TITLE
Nextacloud account in Android is not used

### DISCUSSION THREAD

**Discussion by ildar** [quillpad/quillpad#67] [Ideas]
Greetings!
I see the Nextcloud app created an account in the system. Yet Quillpad doesn't use it requiring a separate login.
Any idea why? Maybe some technical difficulties?

**Comment by jsixface** [comment]
As of right now, quillpad doesn't look into the system accounts. I has the credentials stored locally. 
We may look into integrating with system account in the future.

**Reply by regs01** [reply]
Using Nextcloud account is not just going to be safer and easier, because you wouldn't need to remember it.

However Nextcloud actually mounts as a "drive" into system. So you can just save files right there and Nextcloud app will take care of synchronization by itself. This could also be covered by #26.

**Comment by ildar** [comment]
well, kinda.
But NC app is still unable to do bi-di sync. So that's not really an option.

> Using Nextcloud account is not just going to be safer and easier
exactly safer and easier. You don't pass password between devices, you
don't have to enter any data, just ALLOW using the existing account. That's
what account were really invented for (well, kinda )))

**Comment by ildar** [comment]
2+ years left. Some Android apps started to work with NC account, i.e. logging in without any additional passwords or tokens.
Second thought since then?
